package.json

```diff
@@ -320,7 +320,7 @@
         "@types/node": "^12.8.1",
         "@types/request-promise-native": "^1.0.18",
         "@types/semver": "^7.3.9",
-        "@types/vscode": "^1.66.0",
```

Comment:
After running `vsce package` on this PR (and after pulling changes from `main`) I started getting this:

```
 ERROR  @types/vscode ^1.66.0 greater than engines.vscode ^1.52.0. Consider upgrade engines.vscode or use an older @types/vscode version
```

I guess this problem started when `@types/vscode` was introduced here: https://github.com/fwcd/vscode-kotlin/commit/14664f13c9bf86b42acd80a24fb097c697d8b9b7

I downgraded `@types/vscode` instead of upgrading the engine. Let me know if I should go the other way around.

Reply:
Yes, using the older `@types/vscode` for now seems like the best way forward for now, good catch.